instantout/instantout.go

```diff
@@ -169,7 +169,7 @@ func (i *InstantOut) createHtlcTransaction(network *chaincfg.Params) (
 
 	// Estimate the fee
 	weight := htlcWeight(len(inputReservations))
-	fee := i.htlcFeeRate.FeeForWeight(weight)
+	fee := i.htlcFeeRate.FeeForWeight(lntypes.WeightUnit(weight))
```

Comment:
htlcWeight could return `lntypes.WeightUnit` and skip conversion to `int64` and back steps.

Reply:
Agree, also same for `sweeplessSweepWeight`.